src/generic_ui/scripts/user.ts

```diff
@@ -41,46 +40,61 @@ module UI {
       }
       this.name = profile.name;
       this.imageData = profile.imageData || UI.DEFAULT_USER_IMG;
-      this.isOnline = profile.isOnline;
     }
 
     // Returns two strings, where each matches an array name in model.contacts.
     // Does not use an enum because we need a string value, and typescript
     // enums evaluate to numbers.
```

Comment:
+1 on this. When trying to understand the messages passing between the UI and the app, it was really frustrating to see numbers like 1000, 2001, etc. passing back and forth rather than the actual label of those messages.  If that's what typescript enums are doing, it certainly doesn't make debugging fun.